todo/src/actions.js

```diff
@@ -0,0 +1,40 @@
+export const NEW_TODO = 'NEW_TODO';
+export const COMPLETE_TODO = 'COMPLETE_TODO';
+export const DELETE_TODO = 'DELETE_TODO';
+export const DELETE_COMPLETED = 'DELETE_COMPLETED';
+export const CLEAR_ALL = 'CLEAR_ALL';
+
+export const newTodo = (text) => {
```

Comment:
These all look good 👍 [here](https://github.com/redux-utilities/flux-standard-action#design-goals) is an action pattern you will see used in many places.